### PR TITLE
avatar-component

### PR DESCRIPTION
**Pull request by anshtripathi01**
**_Please Review my Avatar Components_**
Avatar

### CODE REVIEW

File: Components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    justify-content: center;
+    align-items: center;
+   height: 200px;
+   width: 100vh;
+}
+.img-round{
+    border-radius: 50%;
+}
+.avtar{
+    margin-bottom: 0px;
```
Comment (nightlight-wander) [inline Components/avatar.css:13]: for margins and paddings you should use rems ,if you have set html root font-size to 62.5% ,they will work like for 10px 1rem.

**Review by nightlight-wander — COMMENTED** [review]
Class names are good, just some changes which i mentioned in the comment.

File: Components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    margin-left: -100px;
```
Comment (zabihshaik) [inline Components/avatar.css:14]: Better to use shorthand properties for margin (since used left, bottom)

File: Components/avatar.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+}
+.avtar-md{
+    margin-left: 50px;
```
Comment (zabihshaik) [inline Components/avatar.css:17]: Better to use rems instead of px for consistency

**Review by zabihshaik — COMMENTED** [review]
Everything looks good.
Suggestions:
1. Better to use shorthand properties for margin, padding, border, etc
2. Better to use rems instead of px for consistency
3. Add Documentation for the component.

**Pull request merged**